utils/ami-cleanup/ami_cleanup.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import os
+import sys
+import boto3
+
+# Snapshot description filters
+snap_filter_1="Created by CreateImage*for AMI_ID*"
+snap_filter_2="Copied for DestinationAmi AMI_ID*"
+
+# Initialise logging
+logger = logging.getLogger(__name__)
+log_level = os.environ["LOG_LEVEL"] if "LOG_LEVEL" in os.environ else "INFO"
+logger.setLevel(logging.getLevelName(log_level.upper()))
+logging.basicConfig(
+    stream=sys.stdout,
+    format="%(asctime)s %(levelname)s %(module)s "
+           "%(process)s[%(thread)s] %(message)s",
+)
+logger.info("Logging at {} level".format(log_level.upper()))
+
+def get_arguments():
+    parser = argparse.ArgumentParser(description='Delete older AMIs except for N latest.')
+
+    parser.add_argument('--aws-profile', help='AWS profile ID for the account containing AMIs', required=True)
```

Comment:
Just for clarity:
`AWS profile ID for the account containing AMIs` ->
`AWS profile ID for the account that owns the AMIs`